test/Data/ValidationSpec.hs

```diff
@@ -479,16 +479,6 @@ spec = parallel $ do
       let proof = fromVCtx $ optional (Just c) $ \v -> withValue v $ validateField
       proof `shouldBe` Invalid [] (fromList [(['phoneNumber], ["Phone Number cannot be empty."])])
 
-  describe "validateWhen" $ do

```

Comment:
We should test the new functions similar to the way these were done.

Reply:
I'll add these.